packages/blade/src/components/Tooltip/TooltipInteractiveWrapper.web.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import BaseBox from '~components/Box/BaseBox';
+import { makeBorderSize } from '~utils';
+
+const TooltipInteractiveWrapper = styled(BaseBox).attrs(() => {
+  return { tabIndex: -1 };
+})((props) => {
+  return {
+    display: 'inline-block',
+    '&:focus': {
+      borderRadius: makeBorderSize(props.theme.border.radius.medium),
+      // TODO: Replace with focus outline token
```

Comment:
can you replace this token and then remove this Todo?

Reply:
Focus ring tokens aren't refactored out. We have many places where we are doing this. Ideally we should do this from design first.

Reply:
Refer #638